ethcore/src/client/client.rs

```diff
@@ -293,19 +293,14 @@ impl Importer {
 
 				match self.check_and_lock_block(block, client) {
 					Ok(closed_block) => {
-						if self.engine.is_proposal(&header) {
```

Comment:
I don't think this needs to be removed. For clique engine if we don't need proposal, `is_proposal` can always return false.